Python/pytime.c

```diff
@@ -681,21 +755,34 @@ pymonotonic(_PyTime_t *tp, _Py_clock_info_t *info, int raise)
 
 #elif defined(__APPLE__)
     static mach_timebase_info_data_t timebase;
-    uint64_t time;
+    uint64_t ut64;
+    _PyTime_t t;
 
     if (timebase.denom == 0) {
+        uint64_t gcd;
         /* According to the Technical Q&A QA1398, mach_timebase_info() cannot
            fail: https://developer.apple.com/library/mac/#qa/qa1398/ */
         (void)mach_timebase_info(&timebase);
+        gcd = _PyTime_GCD(timebase.numer, timebase.denom);
+        timebase.numer /= gcd;
+        timebase.denom /= gcd;
+        if (timebase.numer < 1 || timebase.denom < 1) {
```

Comment:
When this is possible?

Reply:
It's sanity checks "just in case". In practice, mach_timebase_info() always return (1, 1).

These checks avoid a possible division by zero.